app/Models/Course.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Course extends Model
+{
+    public function user()
+    {
+        return $this->belongsToMany(Users::class, 'courses_users');
```

Comment:
return $this->belongsToMany(User::class); là được

Reply:
khi dùng quan hệ nhiều nhiều nó sẽ tự động tìm đến bảng trung gian, em sợ cách đặt tên bảng trung gian của em không đúng với cách đặt mặc định của nó nên mới để tên bảng trung gian ở đó

Reply:
thế em đọc thêm về belongsToMany để xem trường hợp này có cần không nhé?